src/collections/componentSet.ts

```diff
@@ -5,7 +5,7 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 /* eslint  @typescript-eslint/unified-signatures:0 */
-import { j2xParser } from 'fast-xml-parser';
+import { XMLBuilder } from 'fast-xml-parser';
```

Comment:
worth calling out this was big change from the lib. 
XMLBuilder for js to XML
XMLParser for XML -> js